runtime/lua/vim/ui.lua

```diff
@@ -57,6 +75,13 @@ end
 ---               Called once the user confirms or abort the input.
 ---               `input` is what the user typed.
 ---               `nil` if the user aborted the dialog.
+---
+--- Example:
+--- <pre>
+--- vim.ui.input({ prompt = 'Select value for shiftwidth: ' }, function(input)
```

Comment:
Hmm to me `Select ...` would mean to select among a set of choices, I think it would be more exact to say `Enter ...`

Reply:
https://github.com/neovim/neovim/pull/17251